lib/Option/SanitizerOptions.cpp

```diff
@@ -258,6 +258,22 @@ OptionSet<SanitizerKind> swift::parseSanitizerRecoverArgValues(
   return sanitizerRecoverSet;
 }
 
+bool swift::parseSanitizerAddressUseODRIndicator(
+    const llvm::opt::Arg *A, const OptionSet<SanitizerKind> &enabledSanitizers,
+    DiagnosticEngine &Diags) {
+  if (!A)
+    return false;
```

Comment:
This condition isn't really that valuable, as we shouldn't expect `A` to be null - I would say that this is better served as an `assert`

Reply:
That's true in the current context. The reason I wrote it this way is so someone could write something like.

```
 IRGenOpts.SanitizeAddressUseODRIndicator =
         parseSanitizerAddressUseODRIndicator(
           Args.getLastArg(options::OPT_sanitize_address_use_odr_indicator), 
           Opts.Sanitizers, Diags);
```

However, no code does that today so this isn't a very strong argument.

Reply:
I dropped the condition entirely so we now make the same assumption that the other functions do that `llvm::opt::Arg* A` is not null.

I didn't add an assert because none of the other functions have it.